test/alternator/test_gsi.py

```diff
@@ -1241,8 +1241,7 @@ def test_gsi_list_tables(dynamodb, test_table_gsi_random_name):
 # contain empty string values for the indexed keys. Such values should be
 # skipped while filling the GSI - even if Scylla actually capable of
 # representing such empty view keys (see issue #9375).
-# Reproduces issue #5022 and #9424.
-@pytest.mark.xfail(reason="issue #5022, #9424")
+@pytest.mark.xfail(reason="issue #11567, #9424")
```

Comment:
Thanks for noticing I split 5022. Please leave the "Reproduces ..." comment in addition to the xfail line - the idea is that eventually when we fix this bug, we'll remove the xfail, but the comment about which issue this test reproduces will remain.

Reply:
ok